taskfiles/rpk.yml

```diff
@@ -55,6 +55,7 @@ tasks:
         cmd="$cmd -count=1"
       fi
       if [ '{{.FILTER}}' != '' ]; then
-        cmd="$cmd: -run {{.FILTER}}"
+        cmd="$cmd -run {{.FILTER}}"
       fi
+      echo "$cmd"
```

Comment:
is this `echo` intended to be executed as well?

Reply:
My bad, fixed.